import-service/src/services/s3-management-service.ts

```diff
@@ -50,7 +55,43 @@ export class S3ManagementService implements S3ManagementServiceInterface {
     }
   }
 
-  async moveFiles(objectKeys: string[]): Promise<unknown[]> {
+  //TODO:!!! types
+  async sendProductsToQueue(products: Product[]): Promise<any> {
```

Comment:
This method is not suitable for this s3 service

Reply:
fixed ++